### ISSUE TITLE
Tensorflow-gpu is not using any gpu power

### ISSUE THREAD

**Issue by NUV13102001** [tensorflow/community#122]
Windows 10 x64
GTX 1060 6GB
CUDA - 10.0, 10.1
cuDNN - 7.6.1, 7.6.1
Python - 3.6.5
Anaconda newest version and updated using "conda update conda(-build)"

My installation was as follows:-

Python
Anaconda
Visual Studio - Desktop development with C++
CUDA and cuDNN in chronological order
pip install tensorflow-gpu (ignore installed and upgrade)

I am trying to work on a few simple TensorFlow projects, but over multiple installations, TensorFlow has not used the GPU power it has at its disposal. The specified installation was done after a clean Windows install as well. Tensorflow works in general, but it absolutely does not use any GPU processing power, but occupies 4.5GB of GPU memory. 

I need to get this working as soon as possible. Thanks in advance.

**Comment by NUV13102001** [comment]
Is Linux my only option?

**Comment by PriyanshChaudhary** [comment]
First, have you checked if your CUDA is working in your program?
sometimes anaconda makes an error with CUDA.
first, check on your interpreter if CUDA is being detected.
import torch

>> torch.cuda.current_device()
 0

>> torch.cuda.device(0)
 <torch.cuda.device at 0x7efce0b03be0>

>> torch.cuda.device_count()
> 1

>> torch.cuda.get_device_name(0)
>'GeForce GTX 950M'

>> torch.cuda.is_available()
> True

**Comment by NUV13102001** [comment]
@PriyanshChaudhary Thank you for your answer. I am pretty new, looking to understand some things.

I am currently reinstalling everything. I have not tested your steps yet, but every time a tensorflow device was made, it used the GTX card. It uses the VRAM but not the cores. Why?
Windows >Automatic Update >Python 3.6.6 >Visual Studio Community with C++ Development module >CUDA 10.0 >cuDNN 7.6 >pip install tensorflow-gpu >hello world

After going through the official Tensorflow channel, I discovered that it does not support CUDA 10.1. I am not installing it this time. It however, does not explain my problem.
I got this message while executing a simple hello world program. 
tensorflow/core/grappler/clusters/utils.cc:87] Failed to get device properties, error code: 30
I got the output post this.

I shall try your method now.
